ts/src/test/static/request/yobit.json

```diff
@@ -127,6 +127,18 @@
                         "ETH/USDT"
                     ]
                 ]
+            },
+            {
+                "disabled": true,
+                "description": "all tickers",
+                "method": "fetchTickers",
```

Comment:
@ttodua static tests do not support multiple request at the moment so this test is not very useful

Reply:
and relies on the order of the first request which is not guaranteed